ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6691,3 +6691,11 @@ OAQM:
   - outside_air_co2_concentration_sensor
   implements:
   - MONITORING
+
+ZOC:
+  description: "Occupancy counter monitoring"
+  is_abstract: true
+  uses:
+  - zone_occupant_counter
```

Comment:
How does this sensor work? How is it counting distinct people in the zone?

Reply:
hi, this uses vision (built in camera in the sensor) to count the number of occupance in a room. If allowed and/or needed I can post the link to the sensor model.

Reply:
Thank for the info, please see notes in the telemetry_fields file on naming this field